packages/stats/Dockerfile

```diff
@@ -11,3 +11,11 @@ RUN yarn lerna run build --no-private && yarn workspace @threefold/stats build
 FROM nginx:1.25.3-alpine
 COPY --from=build /app/packages/stats/dist /usr/share/nginx/html
 COPY --from=build /app/packages/stats/nginx /etc/nginx
+RUN mkdir -p /etc/zinit;
+COPY --from=build /app/packages/stats/zinit /etc/zinit
+
+RUN curl --location  https://github.com/threefoldtech/zinit/releases/download/v0.2.10/zinit -o /sbin/zinit && \
```

Comment:
I noticed that the version [v0.2.14](https://github.com/threefoldtech/zinit/releases/tag/v0.2.14) is released, I'm not sure if we have to use it instead